cgi/config.c

```diff
@@ -1779,8 +1779,8 @@ void display_servicedependencies(void) {
 		return;
 		}
 
-	printf("<P><DIV ALIGN=CENTER CLASS='dataTitle'>Service Dependencie%s%s</DIV></P>\n",
-	       (*to_expand == '\0' ? "s" : "s Involving Host "), (*to_expand == '\0' ? "" : html_encode(to_expand, FALSE)));
+	printf("<P><DIV ALIGN=CENTER CLASS='dataTitle'>Service Dependencies%s%s</DIV></P>\n",
+	       (*to_expand == '\0' ? "" : " Involving Host "), (*to_expand == '\0' ? "" : html_encode(to_expand, FALSE)));
```

Comment:
I don't see the value of intentionally using an incomplete word here. Especially when the second half of the substitution was quite willing to use `""`.